juno/assetlist.json

```diff
@@ -1241,6 +1241,28 @@
       "logo_URIs": {
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/spacer.png"
       }            
+    },
+    {
+      "description": "Mille: the 1000th token on osmosis",
+      "type_asset": "cw20",
+      "address": "juno1llg7q2d5dqlrqzh5dxv8c7kzzjszld34s5vktqmlmaaxqjssz43sxyhq0d",
+      "denom_units": [
+        {
+          "denom": "cw20:juno1llg7q2d5dqlrqzh5dxv8c7kzzjszld34s5vktqmlmaaxqjssz43sxyhq0d",
+          "exponent": 0
+        },
+        {
+          "denom": "mile",
+          "exponent": 6
+        }
+      ],
+      "base": "cw20:juno1llg7q2d5dqlrqzh5dxv8c7kzzjszld34s5vktqmlmaaxqjssz43sxyhq0d",
+      "name": "Mille",
+      "display": "mille",
```

Comment:
note sure if the exp 6 denom is meant to be mile or mille, but the 'display' should match the exp-6 denom

Reply:
Meant to be mile. Fixed the typo